src/tests/utils/convert.js

```diff
@@ -97,14 +109,14 @@ describe('convert render functions', () => {
                 type: 'string',
                 required: true,
                 label: 'Username',
-                helpText: '(help text must be at least 10 characters)'
+                helpText: ''
```

Comment:
Does this pass `zapier validate` ? I was initially thinking we didn't add the property at all, but if this still passes validation, I'm ok with it.

Reply:
Nice catch! It does not pass `zapier validate`. Fixed in 01995e7.